### PR TITLE
[self-hosted] Add new Replicated install guide

### PR DESCRIPTION
**Pull request by Pothulapati**
## Description

This PR adds the relevant docs to install Gitpod using replicated. Currently,
This is marked as `beta` by showing a pop-up in the beginning of the doc
notifying users about the same.

Things to consider before merging this:

- [ ] Figure out the right application name, and channels
- [ ] Be clear on the contacting for License step

Signed-off-by: Tarun Pothulapati <tarun@gitpod.io>

## Related Issue(s)

Fixes https://github.com/gitpod-io/website/issues/1743

## How to test

Try out the steps in the Guide.

## Release Notes

```release-note
[self-hosted] Add new Replicated install guide
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1756"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1756"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/installation/index.md [hunk 1]
```diff
@@ -9,12 +9,24 @@ title: Installing Gitpod Self-Hosted
 
 # Install Gitpod Self-Hosted
 
-The current way to install Gitpod on your own Kubernetes cluster is using the Gitpod installer. You will find guides for our supported platforms on the following pages:
+Currently, There are two ways to install Gitpod on a Kubernetes cluster.
+
+## Replicated
+
+> Installing with Replicated is still in Beta. Please feel free to try it out, and [give feedback/bug reports](https://github.com/gitpod-io/gitpod/issues/new/choose)
```
Comment (MrSimonEmms) [inline gitpod/docs/self-hosted/latest/installation/index.md:16]: Suggest changing to "private beta" - at the moment, there's no way of generating a new license to allow people to test it

**Comment by MrSimonEmms** [comment]
Great start. I would strongly suggest that this is not merged yet. We are still awaiting confirmation that Replicated are accepting the work to make a license changeable, which is linked to our requirement to use an anonymous community license.

We also don't have an automated way to create a license.

I would suggest that the wording is changed to say that this is currently in a private beta

File: gitpod/docs/self-hosted/latest/installation/replicated.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+with a adminsitration UI console.
+
+To kick things off, Run
+
+```
+kubectl kots install gitpod-pov/beta
+```
+
+Once you do this, it will ask you the namespace (in which Gitpod should be installed), along with a password for the admin console. You should be able to
+check the `admin-console` where you'll be asked for a Gitpod L;icense. You can also open the Admin UI by running `kubectl kots admin-console -n <namespace>`
```
Comment (corneliusludmann) [inline gitpod/docs/self-hosted/latest/installation/replicated.md:38]: typo

```suggestion
check the `admin-console` where you'll be asked for a Gitpod License. You can also open the Admin UI by running `kubectl kots admin-console -n <namespace>`
```

Comment (MrSimonEmms) [reply gitpod/docs/self-hosted/latest/installation/replicated.md:38]: Can't believe you want me to commit things **WITHOUT** typos!

image

File: gitpod/docs/self-hosted/latest/installation/replicated.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+<script context="module">
+  export const prerender = true;
+</script>
+
+# Install Gitpod Self-Hosted with Replicated
+
+> Installing with Replicated is still in Beta. Please feel free to try it out, and [give feedback/bug reports](https://github.com/gitpod-io/gitpod/issues/new/choose).
+
+[Replicated](www.replicated.com) allows us to package and securely distribute Gitpod to diverse customer environments, without us having to build things specific things for
```
Comment (Cahllagerfeld) [inline gitpod/docs/self-hosted/latest/installation/replicated.md:14]:
```suggestion
[Replicated](https://www.replicated.com) allows us to package and securely distribute Gitpod to diverse customer environments, without us having to build things specific things for
```

**Review by Cahllagerfeld — COMMENTED** [review]
This will fix the preview deployment (hopefully) 😄

**Comment by corneliusludmann** [comment]
> Great start. I would strongly suggest that this is not merged yet.

Let's start with a page that is not linked from the menu/the current pages but only reachable when the URL is known. With that, we can give this to selected customers to try things out. What do you think?

> We are still awaiting confirmation that Replicated are accepting the work to make a license changeable, which is linked to our requirement to use an anonymous community license.

I still don't see it so crucially. I don't think that we will have so many of these cases so quickly and in case, we already know a way to exchange the license even if Replicated does not accept this as a future request. But sure, it would be much more convenient if we get this into Replicated.

> I would suggest that the wording is changed to say that this is currently in a private beta

:100:

File: gitpod/docs/self-hosted/latest/installation/index.md (hunk 1 above)
Comment (MrSimonEmms) [reply gitpod/docs/self-hosted/latest/installation/index.md:16]: There is [a ticket at least](https://github.com/gitpod-io/gitpod/issues/8717)

**Comment by corneliusludmann** [comment]
In perspective, there will be only one way to install Gitpod on the website. Installer docs will live in the repo. That's why an explanation about Replicated & KOTS shouldn't be necessary. Probably the terms “Replicated” and “KOTS” wouldn't have to appear at all (these are more technical details) and instead only the steps necessary for installation would be described. Think more like a white-label solution where the underlying technical foundation isn't of relevance for the customer.

However, a first iteration, especially when we don't link to this page in the first run, doesn't need to be perfect. Just keep this in mind for further iterations.

Overall, still some typos across the text. Probably needs someone that proofreads this with better English skills than me. :grimacing: However, I know that this is just a draft PR that's why I don't want to complain about such small things at this stage. :rofl: Keep going. :rocket:

**Comment by corneliusludmann** [comment]
Closing in favor of https://github.com/gitpod-io/website/pull/1942. Thanks for your work! :+1:

**Pull request closed without merging**